engine/graphics/src/graphics.cpp

```diff
@@ -459,6 +459,9 @@ namespace dmGraphics
         case TEXTURE_FORMAT_RG_BC5:             return 8;
         case TEXTURE_FORMAT_RGBA_BC7:           return 8;
         case TEXTURE_FORMAT_DEPTH:              return 24;
```

Comment:
Not sure it matters, but this might not be correct since we select depth format based on context capabilities